contributors/devel/release/releasenotes.md

```diff
@@ -0,0 +1,94 @@
+# Creating PRs for release notes
+
+This document describes how to create PRs in Kubernetes repos so that the changes can be tracked with the
+Kubernetes release and published in release notes.
```

Comment:
For every unstable release as well (e.g. `v1.8.0-alpha.1`) or is this only for `.0` minor releases like `v1.8.0`?